frontend/src/app/components/wp-custom-actions/date-action/custom-date-action-admin.component.ts

```diff
@@ -30,8 +30,11 @@ import {ApplicationRef, ChangeDetectorRef, Component, ElementRef, OnInit} from '
 import {I18nService} from 'core-app/modules/common/i18n/i18n.service';
 import {DynamicBootstrapper} from 'core-app/globals/dynamic-bootstrapper';
 
+
+export const customDateActionAdminSelector = 'custom-date-action-admin';
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/be03a9ed2bd960efe9bf08e68a4088034cb73cd9/frontend/src/app/components/wp-custom-actions/date-action/custom-date-action-admin.component.ts#L35"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>